plugins/inputs/snmp_trap/snmp_trap_test.go

```diff
@@ -721,71 +722,10 @@ func TestReceiveTrapV3(t *testing.T) {
 				),
 			},
 		},
-		// ordinary v3 coldstart trap SHA auth and no priv
-		{
-			name:      "coldStart authShaNoPriv",
-			secName:   "authShaNoPriv",
-			secLevel:  "authNoPriv",
-			authProto: "SHA",
-			authPass:  "passpass",
```

Comment:
Why is this test removed?

Reply:
Isn't this a duplicate of the test in line 422?

Reply:
Hmm, that might indeed be the case.